tests/unit/EntityMenu.spec.ts

```diff
@@ -456,22 +457,70 @@ describe('Entity Menu - More actions click tests', () => {
 })
 
 describe('Entity Menu - Consent to Continuation click tests', () => {
-  const { assign } = window.location
+  it('displays the Consent to Continuation button', async () => {
+    businessStore.setLegalType(CorpTypeCd.BC_COMPANY)
+    businessStore.setState(EntityState.ACTIVE)
+
+    // mount the component and wait for everything to stabilize
+    const wrapper = mount(EntityMenu, {
+      vuetify,
+      mixins: [{ methods: { isAllowed: () => true } }],
+      propsData: { businessId: 'BC1234567' }
+    })
+
+    await wrapper.find('.menu-btn').trigger('click')
 
+    expect(wrapper.find('#cco-list-item').exists()).toBe(true)
+    expect(wrapper.find('#cco-list-item').text()).toBe('Consent to Continue Out')
+    expect(wrapper.find('#cco-list-item').classes()).not.toContain('v-btn--disabled') // enabled
+
+    wrapper.destroy()
+  })
+})
+
+describe('Entity Menu - Request AGM Extension click tests', () => {
   beforeAll(() => {
-    // mock the window.location.assign function
-    delete window.location
-    window.location = { assign: vi.fn() } as any
+    // override feature flag
+    vi.spyOn(utils, 'GetFeatureFlag').mockImplementation(
+      (flag) => (flag === 'enable-agm-extension'))
```

Comment:
Here is one way to mock the FF. See also line 490.

Reply:
PS - I could have simply mocked the return value, but this is more explicit that it's only True for THIS flag.